TestAutograd.test_multi_grad_all_hooks failure due to missing pybind11 headers

`TestAutograd.test_multi_grad_all_hooks` currently fails on Windows with:

```
FAILED: main.o
cl /showIncludes -DTORCH_EXTENSION_NAME=test_autograd_cpp_node -DTORCH_API_INCLUDE_EXTENSION_H -IC:\miniconda3\envs\torch\Lib\site-packages\torch\include -IC:\miniconda3\envs\torch\Lib\site-packages\torch\include\torch\csrc\api\include -IC:\miniconda3\envs\torch\Lib\site-packages\torch\include\TH -IC:\miniconda3\envs\torch\Lib\site-packages\torch\include\THC -IC:\miniconda3\envs\torch\Include -D_GLIBCXX_USE_CXX11_ABI=0 /MD /wd4819 /wd4251 /wd4244 /wd4267 /wd4275 /wd4018 /wd4190 /wd4624 /wd4067 /wd4068 /EHsc /std:c++17 -c C:\Users\dev-admin\AppData\Local\torch_extensions\torch_extensions\Cache\py312_cpu\test_autograd_cpp_node\main.cpp /Fomain.o
Microsoft (R) C/C++ Optimizing Compiler Version 19.29.30158 for x64
Copyright (C) Microsoft Corporation.  All rights reserved.

C:\miniconda3\envs\torch\Lib\site-packages\torch\include\torch/csrc/Exceptions.h(12): fatal error C1083: Cannot open include file: 'pybind11/pybind11.h': No such file or directory
```


Steps to replicate:
```
conda create -n torch "pytorch==2.5.1" git
conda activate torch
git clone https://github.com/pytorch/pytorch.git
cd pytorch && git checkout a8d6afb511a
python test\test_autograd.py TestAutograd.test_multi_grad_all_hooks
```

--- a/recipe/activate.bat
+++ b/recipe/activate.bat
@@ -46,9 +46,15 @@ set "VSINSTALLDIR=%ProgramFiles(x86)%\Microsoft Visual Studio\2019\Enterprise\"
 )
 
 IF NOT "%CONDA_BUILD%" == "" (
+  :: building packages
   set "INCLUDE=%LIBRARY_INC%;%INCLUDE%"
   set "LIB=%LIBRARY_LIB%;%LIB%"
   set "CMAKE_PREFIX_PATH=%LIBRARY_PREFIX%;%CMAKE_PREFIX_PATH%"
+) else (
+  :: normal environment
+  set "INCLUDE=%CONDA_PREFIX%\Library\include;%INCLUDE%"
+  set "LIB=%CONDA_PREFIX%\Library\lib;%LIB%"
+  set "CMAKE_PREFIX_PATH=%CONDA_PREFIX%\Library;%CMAKE_PREFIX_PATH%"
 )
 
 

--- a/recipe/meta.yaml
+++ b/recipe/meta.yaml
@@ -13,7 +13,7 @@ package:
 
 build:
   skip: True # [not win]
-  number: 5
+  number: 6
 
 outputs:
   - name: vs{{ vsyear }}_{{ cross_compiler_target_platform }}

